src/Error/Debugger.php

```diff
@@ -505,10 +534,13 @@ protected static function _array(array $var, $depth, $indent)
         $vars = [];
 
         if ($depth >= 0) {
+            $mask = (array)static::configInstance('mask');
```

Comment:
~~Could the function always return an array and save the caller a cast?~~ Again ignore me.